azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/presentation/fragment/calling/participantlist/ParticipantListViewModel.kt

```diff
@@ -18,26 +18,19 @@ internal class ParticipantListViewModel(
     private val dispatch: (Action) -> Unit,
 ) {
     private val localUserIdentifier = ""
-    private lateinit var remoteParticipantListCellStateFlow: MutableStateFlow<List<ParticipantListCellModel>>
-    private lateinit var localParticipantListCellStateFlow: MutableStateFlow<ParticipantListCellModel>
-    private val displayParticipantListStateFlow = MutableStateFlow(false)
+    private lateinit var vvmMutableStateFlow: MutableStateFlow<ParticipantListViewViewModel>
+    private lateinit var localParticipantListCellMutableFlow: MutableStateFlow<ParticipantListCellModel>
+    private val displayParticipantListMutableStateFlow = MutableStateFlow(false)
     private lateinit var displayParticipantMenuCallback: (userIdentifier: String, displayName: String?) -> Unit
 
-    fun getRemoteParticipantListCellStateFlow(): StateFlow<List<ParticipantListCellModel>> {
-        return remoteParticipantListCellStateFlow
-    }
-
-    fun getLocalParticipantListCellStateFlow(): StateFlow<ParticipantListCellModel> {
-        return localParticipantListCellStateFlow
-    }
+    val vvmStateFlow: StateFlow<ParticipantListViewViewModel> get() = vvmMutableStateFlow
```

Comment:
Could we rename the stateFlow and elaborate on vvm?

Reply:
It is a viewVeiwModel in the MVVM